doc/optimizers.md

```diff
@@ -484,19 +484,18 @@ Conventional Neural Evolution is an optimizer that works like biological evoluti
 | `double` | **`mutationSize`** | The range of mutation noise to be added. This range is between 0 and mutationSize. | `0.02` |
 | `double` | **`selectPercent`** | The percentage of candidates to select to become the the next generation. | `0.2` |
 | `double` | **`tolerance`** | The final value of the objective function for termination. If set to negative value, tolerance is not considered. | `1e-5` |
-| `double` | **`objectiveChange`** | Minimum change in best fitness values between two consecutive generations should be greater than threshold. If set to negative value, objectiveChange is not considered. | `1e-5` |
 
 Attributes of the optimizer may also be changed via the member methods
-`PopulationSize()`, `MaxGenerations()`, `MutationProb()`, `SelectPercent()`,
-`Tolerance()`, and `ObjectiveChange()`.
+`PopulationSize()`, `MaxGenerations()`, `MutationProb()`, `SelectPercent()`
+and `Tolerance()`.
```

Comment:
I think it could be useful to point out that `CNE` will generate the initial population using a random normal distribution centered at the given initial point.  You could also point out this change in the changelog, noting that it should accelerate convergence.

Reply:
Done, I believe it should be ready to merge now